mage_ai/streaming/sinks/mongodb.py

```diff
@@ -0,0 +1,33 @@
+from dataclasses import dataclass
+from mage_ai.shared.config import BaseConfig
+from mage_ai.streaming.sinks.base import BaseSink
+from typing import Dict, List
+import time
+from pymongo import MongoClient
+
+
+@dataclass
+class MongoDbConfig(BaseConfig):
+    connection_string: str
+    database_name: str
+    collection_name: str
+
+
+class MongoDbSink(BaseSink):
+    config_class = MongoDbConfig
+
+    def init_client(self):
+        self.client = MongoClient(self.config.connection_string)
+        self.database = self.client[self.config.database_name]
+        self.collection = self.database[self.config.collection_name]
+
+    def write(self, data: Dict):
+        self.collection.insert_one({"_source": data})
```

Comment:
Is `"_source"` a special key in MongoDB document or do you just put the data in "_source" for convenience? 
I would suggest checking the type of data. If it's a dict, we just insert the data as it is.

Reply:
Updated it to store dict or if data is not dict store with a key named "data"

Reply:
Could you update the `write` method as well?